proofOC/bookRoom.py

```diff
@@ -86,9 +87,11 @@ def bookRoom(self, roomSize, timeS, roomNum, date):
             print("Incorrect Credentials")
             return False
 
-
+        roomSize = roomSize.strip().lower()
         roomS = "https://charlesstudy.temple.edu/reserve/charles-"
         roomS = roomS+"small" if roomSize=="small" else roomS+"large" if roomSize=="large" else roomS+"error"
```

Comment:
you could make this a one-liner by using f-strings, but i'm not going to block over it. just an idea 💡